corehq/apps/data_dictionary/views.py

```diff
@@ -115,7 +115,7 @@ def _export_data_dictionary(domain):
     )
     export_data = {}
     for case_type in queryset:
-        export_data[case_type.name] = [{
+        export_data[case_type.name or "No Name"] = [{
```

Comment:
translate?

Reply:
None of the other parts of this download are translated, so I went with this. Is it worth translating the whole thing?

Reply:
How about translating this function so the download is done and not worrying about the rest of the UI right now?

Reply:
I think we usually do translate feature flagged content, just not admin-only content like accounting.